### PR TITLE
T gigonz/multi

### PR DESCRIPTION
**Pull request by ggonzalere**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py [hunk 1]
```diff
@@ -166,35 +157,30 @@ def create_build(location,
...
-        raise CLIError("Please specify a build-name")
-    if not yaml and (not multi_arch_tag or not tags):
+    if not yaml and (not multi_arch_tag or not images):
         raise CLIError("Please specify the multiArch-tag and the tags that form the multi-architecture flux")
-    if not build_name:
-        build_name = repository.replace("/", ";") + ":" + multi_arch_tag
-    return _validate_user_credentials(registry_name,
-                                      repository,
+
+    return _validate_user_credentials(registry,
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py]: Revert `registry` back to `registry_name` and register it similar to https://github.com/AppPlatPerf/azure-cli/blob/master/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L49

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py [hunk 2]
```diff
@@ -166,35 +157,30 @@ def create_build(location,
         except FileNotFoundError as e:
             logger.warning("Unable to find file %s", str(e))
 
-def acr_multi_build_definition_create(registry_name,
-                                      repository,
-                                      tags=None,
+def acr_multi_build_definition_create(registry,
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py]: Revert `registry` back to `registry_name` and register it similar to https://github.com/AppPlatPerf/azure-cli/blob/master/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L49

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py [hunk 3]
```diff
@@ -166,35 +157,30 @@ def create_build(location,
...
+                                      images=None,
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py]: This `images` really means dependency or source images. Do you think we can improve the naming?

Comment (ggonzalere) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py]: Hmmm images is also used in the YAML schema, so I thought it was a good idea to keep a standard between the two options.

**Pull request merged**